app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
```

Comment:
- Good job done so far, but as I suggested in the `user` model file, this method should not be implemented here, please fix this 🙏

Reply:
@mckent05 done